src/modules/job-info/guest_watch.c

```diff
@@ -626,15 +626,18 @@ static void guest_namespace_watch_continuation (flux_future_t *f, void *arg)
 
     if (flux_rpc_get (f, &s) < 0) {
         if (errno == ENOTSUP) {
-            /* Guest namespace has been removed.  If we have read no
-             * events in the guest eventlog, assume job was moved into
-             * the main namespace before we began watching in the
-             * guest namespace.
+            /* Guest namespace has been removed.  If we have read
```

Comment:
Commit a3967b7 is mostly overwritten by later commits (squash?)

Reply:
The overwritten comment talks about handling the case of an empty eventlog.  I don't think that can happen.  Eventlogs are created when the first event is appended (perhaps that needs calling out in RFC 18?)

Reply:
Yeah, it is overwritten within this commit series.  I decided to leave the commit series b/c of how some individual issues were fixed along the way.  Could squash.

I assumed there could be cases of `mkdir` without an append.  If that's not possible, could close that issue and squash together.  Let me see about how to re-order squash the commits.

Reply:
> I assumed there could be cases of mkdir without an append.

Did you mean an empty put?

Reply:
> Did you mean an empty put?

What I meant to say was "mkdir and nothing written afterwards".  But yeah, an empty put.